.github/workflows/bot.yml

```diff
@@ -456,15 +457,15 @@ jobs:
       matrix:
         include:
           - scalaProfile: 'scala-2.13'
-            flinkProfile: 'flink1.18'
+            flinkProfile: 'flink1.19'
             sparkProfile: 'spark3.5'
             sparkRuntime: 'spark3.5.0'
           - scalaProfile: 'scala-2.12'
-            flinkProfile: 'flink1.18'
+            flinkProfile: 'flink1.19'
             sparkProfile: 'spark3.5'
             sparkRuntime: 'spark3.5.0'
           - scalaProfile: 'scala-2.12'
-            flinkProfile: 'flink1.18'
+            flinkProfile: 'flink1.19'
```

Comment:
Should this be kept the same as before to cover Flink 1.18?

Reply:
We tend to use the latest Flink release to test.

Reply:
Got it.  My point is this image may need to be updated as well since the Flink version is updated.  Then we should update this image.

Reply:
yeah, that's right.